README.md

```diff
@@ -1,81 +1,69 @@
+You do not need nvim-lspconfig to use neovim's built-in language server client, this repo
+just handles the configuration for you to make it easy
 
 ## Install
 
-- Requires [Nvim HEAD/nightly](https://github.com/neovim/neovim/releases/tag/nightly) (v0.5 prerelease).
-- nvim-lspconfig is just a plugin. Install it like any other Vim plugin, e.g. with [vim-plug](https://github.com/junegunn/vim-plug):
+* Requires [Nvim HEAD/nightly](https://github.com/neovim/neovim/releases/tag/nightly) (v0.5 prerelease). The configs in this repo
+  assume that you are using the latest [Nvim HEAD/nightly build](https://github.com/neovim/neovim/releases/tag/nightly). 
+  Update Nvim and nvim-lspconfig before reporting an issue.
+
+* nvim-lspconfig is just a plugin. Install it like any other Vim plugin, e.g. with [vim-plug](https://github.com/junegunn/vim-plug):
   ```
   :Plug 'neovim/nvim-lspconfig'
   ```
-- Call `:packadd nvim-lspconfig` in your vimrc if you installed nvim-lspconfig to `'packpath'` or if you use a package manager such as minpac.
-
-## Usage
-
-Each config provides a `setup()` function, to initialize the server with
-reasonable defaults and some server-specific things like commands or different
-diagnostics.
-
-```lua
-vim.cmd('packadd nvim-lspconfig')  -- If installed as a Vim "package".
-require'lspconfig'.<config>.setup{name=…, settings = {…}, …}
-```
 
-If you want to add this to your vimrc, you will need to enclose it in a `lua` block.
+Note: if you installed nvim-lspconfig to `'packpath'` you need to call 
+`:packadd nvim-lspconfig` in your vimrc.  This is the case for only 
+some package managers, such as minpac.  You do not need to do this 
+step with vim-plug or packer.
 
+## Quickstart
+1. Install a language server `npm i -g pyright`, and ensure it is on your PATH (`pyright --version` should execute)
+2. Install `nvim-lspconfig`
+3. Add the language server setup to your init.vim
 ```vim
-lua <<EOF
-vim.cmd('packadd nvim-lspconfig')  -- If installed as a Vim "package".
-require'lspconfig'.<config>.setup{name=…, settings = {…}, …}
-EOF
+lua << EOF 
+require'lspconfig'.pyright.setup{}
+EOF 
 ```
+4. Open a file in a directory that triggers the respective language server's root detection. 
+For pyright, this is any directory containing ".git", "setup.py",  "setup.cfg", 
+"pyproject.toml", or "requirements.txt", or child directory thereof. See [CONFIG.md](CONFIG.md) 
+or the source file for each server to check the root trigger.
 
-Find the [config](#configurations) for your language, then paste the example
-given there to your `init.vim`. **All examples are given in Lua,** see `:help
-:lua-heredoc` to use Lua from your init.vim.
+## Usage
 
-Some configs may define additional server-specific functions, e.g. the `texlab`
-config provides `lspconfig.texlab.buf_build({bufnr})`.
+**All examples are given in Lua,** see `:help :lua-heredoc` to use Lua from your init.vim,
```

Comment:
I think this should be `All examples given are in Lua`.

Reply:
'All provided examples`?

Reply:
That sounds better to me.